lib/tool_belt/cherry_picker.rb

```diff
@@ -92,7 +93,11 @@ def write_cherry_pick_log(picks, release)
     end
 
     def log_entry(pick)
-      "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      if pick['bugzilla_id']
+        "#{pick['bugzilla_id']} - #{pick['id']} : [#{pick['revision']}] #{pick['subject']}"
+      else
+        "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      end
```

Comment:
From my testing there was never a 'bugzilla_id' field in any of the picks pulled from Redmine. I injected the bugzilla_id myself when I meshed the bugzilla bug with the corresponding redmine issue. Can you provide an example of a redmine issue that has a bugzilla_id that is actually pulled in with the cherry_picker script?